src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Profiles/index.html.twig

```diff
@@ -25,6 +25,16 @@
 
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
+{% set layoutTitle = 'Profiles'|trans({}, 'Admin.Navigation.Menu') %}
+{% set enableSidebar = true %}
+{% set layoutHeaderToolbarBtn = {
```

Comment:
not really fan of moving business logic into the templates!

Let's keep this in the controllers please :+1:

Reply:
displaying button is not a business logic and in my opinion it belongs in template, no? why add these lines of code in controller and pass it directly to template?

Reply:
the display stays in the template, the data in the controller: separation of concerns 👍

Reply:
Agreed :+1: